boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -2,7 +2,7 @@
  *  FreeLoader
  *
  * Copyright ... ... (See below.)
- * Copyright 2017 Serge Gautherie
+ * Copyright 2017 Serge Gautherie <reactos-git_serge_171003@gautherie.fr>
```

Comment:
No one really cares about emails actually, because they are subject to change at any time.

Reply:
Not true. Our standardized license headers in the Coding Style document on the Wiki request E-Mail addresses.
And most of them live longer than you may assume. I needed E-Mail addresses from all developers for the Git migration, and it turned out that many old ones of former developers are still used on GitHub today.

Reply:
Ah, good to know :)